src/design-system/components/Box/Box.docs.tsx

```diff
@@ -1,116 +1,233 @@
+            <Inset vertical="19px">
+              <Box background="body" marginTop="-19px">
+                <Placeholder />
+              </Box>
+            </Inset>
+            <Box background="body" marginBottom="-19px">
+              <Placeholder />
+            </Box>
           </Stack>
-        </Columns>
-      ),
+        ),
     },
     {
-      name: 'With animation',
-      Example: () => {
-        const offsetRatio = useSharedValue(0);
-        const containerWidth = useSharedValue(0);
-        const boxSize = 100;
-
-        const sizeStyles = useMemo(
-          () => ({
-            width: boxSize,
-            height: boxSize,
-          }),
-          [boxSize]
-        );
+      name: 'Border radius',
+      description: [
+        <Docs.Text key="">
+          To apply a border radius, supply the{' '}
+          <Docs.Code>borderRadius</Docs.Code> prop with a numerical pixel value.
+          The system also supports border radius on directional or specific
+          corners as seen below.
+        </Docs.Text>,
+      ],
+      Example: () =>
+        source(
+          <Stack space="24px">
+            <Box background="accent" borderRadius={16} padding="19px" />
+            <Box background="accent" borderLeftRadius={16} padding="19px" />
+            <Box background="accent" borderRightRadius={16} padding="19px" />
+            <Box background="accent" borderTopRadius={16} padding="19px" />
+            <Box background="accent" borderBottomRadius={16} padding="19px" />
+            <Box
+              background="accent"
+              borderBottomLeftRadius={16}
+              padding="19px"
+            />
+            <Box
+              background="accent"
+              borderBottomRightRadius={16}
+              padding="19px"
+            />
+            <Box background="accent" borderTopLeftRadius={16} padding="19px" />
+            <Box background="accent" borderTopRightRadius={16} padding="19px" />
+          </Stack>
+        ),
+    },
+    {
+      name: 'Widths',
+      description: [
+        <Docs.Text key="">
+          To apply a width, fractional values (e.g. <Docs.Code>1/3</Docs.Code>)
+          or <Docs.Code>full</Docs.Code> can be supplied to the{' '}
+          <Docs.Code>width</Docs.Code> prop.
+        </Docs.Text>,
+      ],
+      Example: () =>
+        source(
+          <Stack space="12px">
+            <Box background="accent" padding="24px" width="1/3" />
+            <Box background="accent" padding="24px" width="full" />
+          </Stack>
+        ),
+    },
+    {
+      name: 'Composition',
+      description: [
+        <Docs.Text key="">
+          There may be cases where you want to compose{' '}
+          <Docs.Code>Box</Docs.Code>, such as animated components. To render
+          another element instead, you can use the <Docs.Code>as</Docs.Code>{' '}
+          prop.
+        </Docs.Text>,
+        <Docs.CodeBlock
+          code={`const offsetX = useSharedValue(0);
```

Comment:
Might be worth using [dedent](https://www.npmjs.com/package/dedent) to make this a bit nicer.

Reply:
Ooo, yes!